CMSgov/design-system
# [WNMGDS-1466] icon title updates

## Summary

* Updated the `SvgIcon` to have a `hideTitle` prop and give consuming teams the ability to not render the title prop if they have another accessible label.
* Updated the `TooltipIcon` component to have a screen-reader accessible, optional label other than the title, so that it doesn't conflict with tooltip content
* Updated the tooltip icon tests to use testing library

## How to test

`yarn storybook`. Navigate to Components > Tooltip. Hover over the tooltip icon and confirm that the small grey 'title' no longer appears

## Outstanding questions
* Making this change to SvgIcon changed some of our internal components that are using icons. I _think_ most of them are okay being hidden to screen readers, but I need a second opinion
